apps/src/code-studio/progressRedux.js

```diff
@@ -709,6 +709,7 @@ export const progressionsFromLevels = levels => {
   let currentProgression = {
     start: 0,
     name: levels[0].progression || levels[0].name,
+    display_name: levels[0].progressionDisplayName || levels[0].name,
```

Comment:
nit: please use camel case for variables names which exist only in JS: `displayName`